neumai-tools/neumai_tools/PipelineCollection/PipelineCollection.py

```diff
@@ -1,4 +1,5 @@
 from typing import List
+from scipy.spatial.distance import cosine
```

Comment:
Thoughts on using numpy instead of scipy given that we are already using numpy in other places. If using scipy, lets add to requirements.

Reply:
Yes that would be better, thanks, made the changes.